app/javascript/src/editable_components.js

```diff
@@ -761,38 +766,24 @@ class EditableCollectionItemInjector {
   }
 }
 
-
-class EditableCollectionItemRemover {
-  constructor(editableCollectionItem, editableCollectionFieldComponent, config) {
-    var conf = mergeObjects({}, config);
-    var text = mergeObjects({ itemRemove: 'remove' }, config.text);
-    var $node = $(createElement("button", text.itemRemove, conf.classes));
-
-    $node.data("instance", this);
-    $node.addClass("EditableCollectionItemRemover");
-    $node.attr("type", "button");
-    $node.on("click.EditableCollectionItemRemover", function(e) {
-      e.preventDefault();
-      editableCollectionItem.remove();
-    });
-
-    // Close on ENTER || SPACE
-    $node.on("keydown.EditableCollectionItemRemover", function(e) {
-      e.preventDefault();
-      if(e.which == 13 || e.which == 32) {
-        editableCollectionItem.remove();
+function createEditableCollectionItemMenu(item, config) { 
+  var template = $("[data-component-template=EditableCollectionItemMenu]");
```

Comment:
Inclined to think this this kind of of thing would be nicer passed in from a view controller through configs, but it's still better than the direct HTML inside JS stuff that has now been removed elsewhere in this file.